core/src/main/java/jenkins/websocket/WebSockets.java

```diff
@@ -68,12 +68,20 @@ public static HttpResponse upgrade(WebSocketSession session) {
         return (req, rsp, node) -> {
             try {
                 session.handler = provider.handle(req, rsp, new Provider.Listener() {
+                    Object providerSession;
```

Comment:
Visibility could be decreased:

```suggestion
                    private Object providerSession;
```

Generally desirable to keep visibility as low as possible to make future changes easier.

Reply:
Not necessary in this case because it is an anonymous inner class.